system/assets/js/admin/specific/auditTrail/index/auditPagination.js

```diff
@@ -0,0 +1,56 @@
+( function( $ ){
```

Comment:
We can move this to `/js/admin/presidecore` (name it preside.load.more.js). Its pretty small and I'm sure we will use it again.